src/bptl/work_units/zgw/tasks/zaak_relations.py

```diff
@@ -310,6 +311,44 @@ def perform(self) -> Optional[dict]:
             request_headers=headers,
         )
 
+        try:
+            bijdrage_aard_omgekeerde_richting = check_variable(
+                variables, "bijdrageAardOmgekeerdeRichting", empty_allowed=True
+            )
+            if (
+                bijdrage_aard_omgekeerde_richting
+                and bijdrage_aard_omgekeerde_richting not in AardRelatieChoices.values
+            ):
+                raise ValueError(
+                    f"Unknown 'bijdrageAardOmgekeerdeRichting': '{bijdrage_aard_omgekeerde_richting}'"
+                )
+        except MissingVariable:
+            # To avoid having to edit BPMN models - default of bijdrage_aard_omgekeerde_richting
+            # is "onderwerp" if it isn't explicitly given in the variables.
+            bijdrage_aard_omgekeerde_richting = AardRelatieChoices.onderwerp
+
+        # Relating of secondary zaken to their main zaak.
+        if (
+            bijdrage_aard != AardRelatieChoices.onderwerp
```

Comment:
In the issues there is nothing about the reverse relation should be added only if `aardRelatie` = "onderwerp". It's specified that FB should be able to specify both types of relations. Could you please double check it?

Reply:
I figured it doesn't make sense to add it if the `bijdrage_zaak == "onderwerp"` right? I.e., the main zaak's onderwerp can't be the related zaak if the related zaak's onderwerp is the main zaak. Or am I mistaken?

I guess it should be mutually exclusive, not necessarily only when the `aardrelatie == "onderwerp"`.

Reply:
Afaik there is no check for it in the Zaken API, so technically you can add any combination of `aardrelatie`. Logically I think you are right, but functional beheers should be ok with it, or it would be an unpleasant surprise for them.

Reply:
It is logically correct even though technically it's possible. Two cases being the subject of each other does not make sense, so we should avoid getting to that situation.